### PR TITLE
Merge release-8.0.2 into develop-8.0

### PR DESCRIPTION
**Pull request by peter-b**
Trivial conflict in `version`.

### CODE REVIEW

File: ide [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 372252ac065560dec60c74e823a228eb2925b5ad
+Subproject commit 504860b26826acade8955ca49377993913358eec
```
Comment (livecodeali) [inline ide]: I don't think this submodule pointer update is correct:
https://github.com/livecode/livecode-ide/commits/develop-8.0
https://github.com/livecode/livecode-ide/commits/504860b26826acade8955ca49377993913358eec

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 823ee5d

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 823ee5d07511ef7ff237302d8b1262f08f50a270

**Comment by livecode-vulcan** [comment]
😎 test success 823ee5d07511ef7ff237302d8b1262f08f50a270
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2660)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2670)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2655)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2671)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2677)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2683)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2695)

**Comment by livecodefraser** [comment]
Failed fetching from the mergExt download URL it seems...

**Comment by livecodefraser** [comment]
I'm not sure what's going on with the mergExt fetching. The latest builds are uploaded to the server but the failure is still occurring.

**Pull request merged**